### DISCUSSION TITLE
Locally Encrypt notes on disk

### DISCUSSION THREAD

**Discussion by vHanda** [GitJournal/GitJournal#689] [Feature Requests]
A user has requested that the notes never be stored on disk in an unencrypted manner. This means having a kind of virtual file system which GitJournal accesses.

To clarify: The notes would not be committed to the git repo in an encrypted manner. This encryption would only be local to the device and would protect you in case someone gets access to your device.

Standard notes has something [similar](https://standardnotes.com/help/79/how-does-standard-notes-encrypt-data-on-my-device).

On the other hand, many people already encrypt the file system in at least Android. Is this second level really required?

---
> Please vote on this issue with a 👍 reaction. It's one of the primary mechanisms used to prioritize issues. Please also subscribe to the issue if you're interested, as that way I can discuss possible implementations and share mockups.

**Comment by Dieterbe** [comment]
That would mean you need to enter some sort of credentials when you want to use gitjournal?

IMHO you should avoid other people getting access to your device. If someone gets access into your running device they also have access to your pictures, email, etc etc.
So use device lock/unlock (and encrypted disk for the paranoid) and protect everything all with 1 mechanism

**Comment by vHanda** [comment]
> IMHO you should avoid other people getting access to your device. If someone gets access into your running device they also have access to your pictures, email, etc etc. So use device lock/unlock (and encrypted disk for the paranoid) and protect everything all with 1 mechanism

I agree. Encrypting your entire disk is much easier, and this is therefore not a priority right now.
